go/libkb/secret_store.go

```diff
@@ -111,24 +111,55 @@ func NewSecretStore(g *GlobalContext, username NormalizedUsername) SecretStore {
 }
 
 func GetConfiguredAccounts(m MetaContext, s SecretStoreAll) ([]keybase1.ConfiguredAccount, error) {
-
 	currentUsername, allUsernames, err := GetAllProvisionedUsernames(m)
 	if err != nil {
 		return nil, err
 	}
-
 	if !currentUsername.IsNil() {
 		allUsernames = append(allUsernames, currentUsername)
 	}
 
 	accounts := make(map[NormalizedUsername]keybase1.ConfiguredAccount)
-
 	for _, username := range allUsernames {
 		accounts[username] = keybase1.ConfiguredAccount{
 			Username:  username.String(),
 			IsCurrent: username.Eq(currentUsername),
 		}
 	}
+
+	// Get the full names
+
+	uids := make([]keybase1.UID, 0, len(allUsernames))
+	for _, username := range allUsernames {
```

Comment:
it's more efficient to do 

    uids := make([]keybase1.UID, len(allUsernames)
    for i, username := range allUsernames {
        // ...
        uids[i] = uid
    }

Reply:
Just for fun I ran a test, and it is indeed 7-8% faster!